pkg/cluster/cluster.go

```diff
@@ -46,6 +46,11 @@ const (
 	DefaultPushPullInterval = 5 * time.Second
 	DefaultGossipInterval   = 5 * time.Second
 	DefaultRefreshInterval  = 60 * time.Second
+
+	// peer's network types
+	DefaultLocalPeerType = "local"
```

Comment:
Personally, I think `Default Prefix is little bit verbose, but it's up to you.

It is a "base" configuration that then:
```
DefaultPushPullInterval, DefaultGossipInterval, DefaultRefreshInterval
```
are applied, but `Default` prefix is not helping to understand this.

Reply:
Also wonder if in that case we should not remove these 3 defaults (pullInterval, gossipInterval, RefreshInterval) and if any of this flag is not explicitly specified, we should use default from, Lan, local etc. What do you think?

Reply:
yeah, will update, should use defaults related network type. Probably, further, we want to use all this knobs (`memberlist.Config`) as arguments or in configurations, tho.

Reply:
Not sure about this. Let's allow only these critical ones for now (: We can add more later.